functional-samples/cookbook.sidepanel-open/README.md

```diff
@@ -0,0 +1,25 @@
+# Opening the side panel through a user interaction
```

Comment:
@sebastianbenz Do you think I'm demonstrating too many use cases in one sample? Do you think I should narrow it down to one?

Reply:
Nope, I think this is perfect